x-pack/test/functional/apps/canvas/reports.ts

```diff
@@ -198,7 +198,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           "
         `);
 
-        expect(res.get('content-length')).to.be('20725');
+        const contentLength = parseInt(res.get('content-length'), 10);
+        expect(contentLength >= 20725 && contentLength <= 20726).to.be(true); // contentLength can be between 20725 and 20726
```

Comment:
nit: should these be documented constants, or is this specific to this test?

Reply:
This is specific to this test - it's essentially a snapshot of the content length of the downloaded PDF file.